packages/frontend/src/components/common/Filters/FilterInputs/DateFilterInputs.tsx

```diff
@@ -170,24 +170,13 @@ const DateFilterInputs = <T extends ConditionalRule = DateFilterRule>(
             }
 
             if (isTimestamp) {
-                const parsedTimestampToUtc = rule.values
-                    ? dayjs(
-                          parseTimestamp(
-                              formatTimestamp(
-                                  rule.values[0],
-                                  TimeFrames.MILLISECOND,
-                              ),
-                              TimeFrames.MILLISECOND,
-                          ),
-                      ).utc()
-                    : null;
+                // For display only
+                const displayFormat = 'YYYY-MM-DD HH:mm:ss (Z)';
```

Comment:
Note that this type of (non-standard) format is dangerous. If you try to parse a string with this format with moment of dayjs, you will not get a correct time because the timezone will be dropped. 

I kept it anyway for compatibility, but it is only for view-time.